src/main/java/beam/utils/gtfs/MATSim2GTFS.java

```diff
@@ -143,8 +143,8 @@ public static void main(String[] args) {
                         gtfsStopTime.trip_id = gtfsTrip.trip_id;
                         gtfsStopTime.stop_sequence = stopWithinTrip;
                         gtfsStopTime.stop_id = trs.getStopFacility().getId().toString();
-                        gtfsStopTime.arrival_time = (int) (firstDepartureTime + trs.getArrivalOffset());
-                        gtfsStopTime.departure_time = (int) (firstDepartureTime + trs.getDepartureOffset());
+                        gtfsStopTime.arrival_time = (int) (firstDepartureTime + trs.getArrivalOffset().orElse(0));
```

Comment:
This is a breaking change as it had previously returned a `Double.NEGATIVE_INFINITY`, but it feels like now that we have this `Optional` mechanism we should treat empty as 0 instead of tanking the arrival time. That seems more correct, but should be verified.